docs/sources/k6/next/using-k6/secret-source/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Secret Source
+description: 'Secret source are used so k6 can get and use secrets in a secure way'
+weight: 14
+---
+
+# Secret source
+
+Secret sources are a way for k6 to acquire secrets to be used with k6.
```

Comment:
It would be good to mention the advantages to use a secret in k6 instead of using the other forms (envs, files, variables, etc...). I expect not everyone knows the various implications of not doing it (e.g. archives).

Reply:
@mstoykov It would be still good to have it, but it's not blocking.